ansible_events_ui/db/migrations/versions/20220816_2afab2d669fe_rbac.py

```diff
@@ -0,0 +1,71 @@
+"""Adds user roles.
+
+Revision ID: 2afab2d669fe
+Revises: 42461b4ec88d
+Create Date: 2022-08-16 08:48:58.150715
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import UUID
+
+# revision identifiers, used by Alembic.
+revision = "2afab2d669fe"
+down_revision = "09d9de4b4624"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "role",
+        sa.Column("id", sa.Integer),
+        sa.Column("name", sa.String),
+        sa.PrimaryKeyConstraint("id"),
```

Comment:
Primary key and foreign key constraints should be named explicitly according to the naming schema. Normally `alembic revision --autogenerate` will generate names for you, but since the migration had been created before the naming schema was applied.
Please refer to the initial migration for an example.

```
sa.PrimaryKeyConstraint("id", name=op.f("pk_role")),
```